packages/forest_admin_agent/lib/forest_admin_agent/routes/capabilities/collections.rb

```diff
@@ -0,0 +1,45 @@
+require 'json'
+module ForestAdminAgent
+  module Routes
+    module Capabilities
+      class Collections < AbstractRoute
+        include ForestAdminDatasourceToolkit::Schema
+
+        def setup_routes
+          add_route('forest_capabilities_collections',
+                    'post',
+                    '/_internal/capabilities',
+                    ->(args) { handle_request(args) })
+
+          self
+        end
+
+        def handle_request(args = {})
+          @datasource = ForestAdminAgent::Facades::Container.datasource
+          collections = args[:params]['collectionNames'] || @datasource.collections.keys
```

Comment:
We said the frontend always sent `collectionNames`, I thought we said to **not** compute filters over all collections if `collectionNames` is missing. Am I wrong ?

Reply:
fixed !